roles/etcd_client_certificates/meta/main.yml

```diff
@@ -14,3 +14,4 @@ galaxy_info:
   - system
 dependencies:
 - role: etcd_common
+- role: delegated_serial_command
```

Comment:
Yeah, remember how we cut down on certain dependencies and added checks like ensuring the etcd ca exists in `etcd_client_certificates` to cut down on the inflated run time w/ 2.1? We can undo that but I expect there are still quite a few 2.1 users.

7ab97920 - Role dep migration for master/node/hosted.
3bd5ae2 - `etcd_ca` removed from etcd_client_certificates dep here.